worker/src/zimfarm_worker/task/worker.py

```diff
@@ -727,24 +844,45 @@ def check_files(self):
                 self.docker,
                 self.checker.name,  # pyright: ignore[reportArgumentType]
             ).strip()
+
+            zimcheck_info = get_zim_info(
+                self.task_workdir
+                / zim_file  # pyright: ignore[reportUnknownArgumentType]
+            )
+
             try:
                 zimcheck_result = ujson.loads(zimcheck_log)
             except Exception as exc:
+                # Failed to parse JSON, send raw log
                 zimcheck_result = None
                 logger.warning(f"Failed to parse zimcheck output: {exc}")
-            else:
-                zimcheck_log = None
```

Comment:
Why not setting zimcheck_log to None anymore? Doesn't it means we will upload both raw and parsed logs?